frame/generic-asset/src/lib.rs

```diff
@@ -178,25 +178,15 @@ mod tests;
 pub use self::imbalances::{NegativeImbalance, PositiveImbalance};
 
 pub trait Trait: frame_system::Trait {
-	type Balance: Parameter
-		+ Member
-		+ AtLeast32Bit
-		+ Default
-		+ Copy
-		+ MaybeSerializeDeserialize
-		+ Debug;
+	type Balance: Parameter + Member + AtLeast32BitUnsigned + Default + Copy +
```

Comment:
can probably fit even in one line?

Reply:
not in 100 character but fits in 120, anyway I just reorganize slightly